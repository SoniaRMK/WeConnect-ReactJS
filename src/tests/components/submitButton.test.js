import {shallow} from 'enzyme';
import React from 'react';
import SubmitButton from '../../components/helper/submitButton';


describe('SubmitButton component',() => {

  it('should match snapshot',() =>{
      const wrapper = shallow(<SubmitButton/>)
      expect(wrapper).toMatchSnapshot();
  })
})
