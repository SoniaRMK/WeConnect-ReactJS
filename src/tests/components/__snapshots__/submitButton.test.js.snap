// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton component should match snapshot 1`] = `
<div
  className="submitButton"
>
  <button
    className="btn btn-info"
    style={
      Object {
        "float": "left",
      }
    }
    type="submit"
  >
    Submit
  </button>
</div>
`;
